### PR TITLE
[REC-2024] Add new organizer and close the registration

### PR DESCRIPTION
**Pull request by yagonobre**
Add new organizer and close the registration

### CODE REVIEW

File: data/events/2024/recife/main.yml [hunk 1]
```diff
@@ -66,6 +64,12 @@ team_members: # Name is the only required field for team members.
     bio: ""
     linkedin: "https://www.linkedin.com/in/fabioqcorreia"
 
+  - name: "Henrique Cordeiro"
+    employer: "Avantsoft"
+    image: "henrique-cordeiro.png"
+    bio: ""
+    linkedin: "https://www.linkedin.com/in/henrique-cordeiro-pereira/"
+
```
Comment (toshywoshy) [inline data/events/2024/recife/main.yml:72]: You need to email info@ to change or add organizers

File: static/events/2024-recife/organizers/henrique-cordeiro.png
Comment (toshywoshy) [inline static/events/2024-recife/organizers/henrique-cordeiro.png:1]: We moved all images to the assets folder
Please move to `assets/events/2024-recife/organizers`

**Review by toshywoshy — CHANGES_REQUESTED** [review]

File: data/events/2024/recife/main.yml (hunk 1 above)
Comment (yagonobre) [reply data/events/2024/recife/main.yml:72]: Done!

File: static/events/2024-recife/organizers/henrique-cordeiro.png
Comment (yagonobre) [reply static/events/2024-recife/organizers/henrique-cordeiro.png:1]: Done!

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**